build.gradle

```diff
@@ -75,16 +75,16 @@ licenseFile = rootProject.file('LICENSE.txt')
 noticeFile = rootProject.file('NOTICE.txt')
```

Comment:
Please turn off your auto formatter, it makes it harder to review your changes.  If you'd like a format to be followed, lets see about enforcing it via a checkstyle rule for consistency in the codebase in a separate PR

Reply:
Oops, I did not even know it was on. Sorry.